src/Microsoft.ML.Data/Transforms/ConcatTransform.cs

```diff
@@ -497,7 +497,7 @@ private void GetSlotNames(int iinfo, ref VBuffer<DvText> dst)
             }
         }
 
-        public const string Summary = "Concatenates two columns of the same item type.";
+        public const string Summary = "Concatenates one or more columns of the same item type.";
```

Comment:
I am not sure if there is a standard when it comes to pluralizing the word after the phrase "one or more"

If this is wrong, I'll change this to one or more column.

Reply:
Interesting question. Never quite thought about it before. But if someone were to say to me, "I saw one or more car drive over the bridge" that *feels* wrong, so I'd be inclined to agree with "columns" (that is, your current description) not "column."

[Here's something that more or less directly talks about this question.](https://english.stackexchange.com/questions/13284/which-is-correct-one-or-more-is-or-one-or-more-are) It sounds like the majority of usage supports your phrasing of it.

If you wanted to avoid the controversy altogether, you could phrase it, "one or more than one column" instead of "one or more columns" but I'd consider that going too far. It's good the way it is I think.

---
In reply to: [199040665](https://github.com/dotnet/machinelearning/pull/448#discussion_r199040665) [](ancestors = 199040665)